packages/components/scss/bootstrap_imports.scss

```diff
@@ -0,0 +1,49 @@
+/*!
+ * Bootstrap v4.6.2 (https://getbootstrap.com/)
+ * Copyright 2011-2022 The Bootstrap Authors
+ * Copyright 2011-2022 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/main/LICENSE)
+ */
+
+/* 
+ * Deephaven manually imports only the necessary bootstrap files.
+ * See `packages/components/scss/bootstrap.scss` for the original list of imports.
```

Comment:
404

Reply:
the original import exists in the node_modules folder for bootstrap.

Reply:
fixed